Formula/quick-lint-js.rb

```diff
@@ -1,21 +1,11 @@
 class QuickLintJs < Formula
   desc "Find bugs in your JavaScript code"
   homepage "https://quick-lint-js.com/"
-  url "https://c.quick-lint-js.com/releases/2.6.0/source/quick-lint-js-2.6.0.tar.gz"
-  sha256 "6fd402e1d0743adb9e862532e25b2be09f637d4c45cb964251ac0f52a1eb5d5c"
+  url "https://c.quick-lint-js.com/releases/2.7.0/source/quick-lint-js-2.7.0.tar.gz"
+  sha256 "722a41b7c5521b2dd483b86ece20dfded6a13b8c8894e9ef2902f2a70c4735ff"
   license "GPL-3.0-or-later"
-  revision 3
   head "https://github.com/quick-lint/quick-lint-js.git", branch: "master"
 
-  bottle do
-    sha256 cellar: :any,                 arm64_monterey: "12108ca1f7680f37d5554e167688cc8b3dcc45d3218a5ead3aaa3368d00933b6"
-    sha256 cellar: :any,                 arm64_big_sur:  "b58448d800e585607b1601df169442dbfb61bc533f38f01f294453571fd7615e"
-    sha256 cellar: :any,                 monterey:       "69cd1977b006efe350c56d4895547f7302141a75e49f056c9b904cd78a01af2e"
-    sha256 cellar: :any,                 big_sur:        "eca1f6d79c2d65dce07d27cc2a63b5d2646f728a31e37c27dd5e3757b71aed0e"
-    sha256 cellar: :any,                 catalina:       "63625efae03df8562639e75cd354169d60909da71f5f817fb149e65332c37589"
-    sha256 cellar: :any_skip_relocation, x86_64_linux:   "549dbff26a0c88ac3f5f986ccdf570976982171195a4a928eb460bfc546bb498"
-  end
-
```

Comment:
dont remove this, this would be handled by @BrewTestBot

Reply:
Leaving these lines in causes `brew install quick-lint-js` to fail:
```
==> Downloading https://ghcr.io/v2/homebrew/core/quick-lint-js/manifests/2.7.0
curl: (22) The requested URL returned error: 404

Error: quick-lint-js: Failed to download resource "quick-lint-js_bottle_manifest"
Download failed: https://ghcr.io/v2/homebrew/core/quick-lint-js/manifests/2.7.0
```
Should I still keep the bottle lines?

Reply:
> Leaving these lines in causes brew install quick-lint-js to fail:

That's only if you run `brew install <name>` instead of `brew install --build-from-source <name>`. The bottle block should stay otherwise CI can't merge this.